Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,32 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Logger.Tests
+﻿namespace Logger.Tests
 {
+    using System;
+    using System.IO;
+    using System.Linq;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
     [TestClass]
     public class FileLoggerTests
     {
-        
+        [TestMethod]
+        public void Log_GivenValidFileLoggerWrites()
+        {
+            // Arrange
+            string filePath = Path.GetFileName("Test.txt");
+            string loggerName = "TestingLogger";
+            string loggerMessage = "Testing Message On The Logger";
+            LogLevel logLevel = LogLevel.Debug;
+            FileStream fileStream = new FileStream(filePath, FileMode.Append);
+
+            // Act
+            LogFactory.ConfigureFileLogger(filePath);
+            FileLogger fileLogger = LogFactory.CreateLogger(loggerName);
```

Comment:
```suggestion
            FileLogger fileLogger = new FileLogger(filePath);
```
We are not testing the functionality of the `LogFactory` here, you want to keep your unit tests specific to the components you are testing. In this case we are testing the FileLogger, so it would be better to directly instantiate a new FileLogger rather than relying on the CreateLogger method which could cause unrelated failures.